src/popup.html

```diff
@@ -19,47 +19,124 @@
+              data-event-label="panel-terms-of-service" data-i18n-message-id="popupSettingsTermsOfService"></a>
+            <!-- Report Issue Panel Prompt -->
+            <a class="setting-label setting-link settings-report-issue i18n-content" data-event-action="click"
+              data-event-label="panel-report-issue" data-i18n-message-id="popupSettingsReportIssue"></a>
           </li>
         </ul>
       </settings>
 
+      <!-- Report Issue Panel -->
+      <report-issue>
+        <div class="settings-header">
+          <button class="close-settings settings-report-issue-return i18n-attribute" data-i18n-attribute="title"
+            data-i18n-message-id="popupSettingsReturn"></button>
+          <h1 class="settings-hl i18n-content" data-i18n-message-id="popupSettingsReportIssue"></h1>
+        </div>
+        <form class="report-issue-content">
+          <!-- Report URL -->
+          <div class="report-section">
+            <label class="report-label i18n-content" for="issue_on_domain" data-i18n-attribute="title"
+              data-i18n-message-id="popupReportIssueSiteLabelHeadline"></label>
+            <input type="text" id="issue_on_domain" name="issue_on_domain" value="" />
+          </div>
+          <div class="report-section">
+            <!-- Select issue checkboxes -->
+            <label class="report-label i18n-content" data-i18n-message-id="popupReportIssueSituationHeadline"></label>
+            <ul>
+              <li>
+                <!-- Case One -->
+                <input type="checkbox" id="email_mask_not_accepted" name="email_mask_not_accepted" />
+                <label class="i18n-content" for="email_mask_not_accepted"
+                  data-i18n-message-id="popupReportIssueCaseOne"></label>
+              </li>
+              <li>
+                <!-- Case Two -->
+                <input type="checkbox" id="add_on_visual_issue" name="add_on_visual_issue" />
+                <label class="i18n-content" for="add_on_visual_issue"
+                  data-i18n-message-id="popupReportIssueCaseTwo"></label>
+              </li>
+              <li>
+                <!-- Case Three -->
+                <input type="checkbox" id="email_not_received" name="email_not_received" />
+                <label class="i18n-content" for="email_not_received"
+                  data-i18n-message-id="popupReportIssueCaseThree"></label>
+              </li>
+              <li>
+                <!-- Case Four -->
+                <input type="checkbox" id="issue-case-other" name="issue-case-other" />
+                <label class="i18n-content" for="issue-case-other"
+                  data-i18n-message-id="popupReportIssueCaseOther"></label>
+              </li>
+              <li>
+                <!-- 'Other' option input field -->
+                <input class="report-other-field is-hidden" type="text" name="other_issue"
+                  id="other_issue" name="other_issue" />
```

Comment:
suggestion (non-blocking): Duplicate `name` attribute. 
```suggestion
                <input class="report-other-field is-hidden" type="text" name="other_issue"
                  id="other_issue" />
```